argon2.yaml

```diff
@@ -20,9 +20,15 @@ pipeline:
       tag: ${{package.version}}
       expected-commit: 62358ba2123abd17fccf2a108a301d4b52c01a7c
 
+  - uses: patch
+    with:
+      # The patch source is https://github.com/NixOS/nixpkgs/blob/e61356b9863f71e7b965f9b8ac3820e7d71af1a0/pkgs/development/libraries/libargon2/default.nix#L21
```

Comment:
Let's instead reference the merged PR so it's a bit clearer https://github.com/P-H-C/phc-winner-argon2/pull/277